src/main/groovy/com/github/lkishalmi/gradle/gatling/GatlingPlugin.groovy

```diff
@@ -83,7 +82,8 @@ class GatlingPlugin implements Plugin<Project> {
                     args "-m"
                     args "-bf", "${project.sourceSets.gatling.output.classesDir}"
                     args "-s", simu
-                    args gatlingExt.gatlingArgs(project)
+                    args "-df", "${project.sourceSets.gatling.output.resourcesDir}"
```

Comment:
I'm just asking, that is it a good idea to use the processed resource folder output here where these resources are poured together regardless from their origin?

1. I see the point that this is the processed resources folder.
2. I do not see why don't we use the separate the data and the bodies source directories.
  (I'm not a big Gatling guy, I do not know why they separate these dirs, but if they are having reasons for that, we need to follow that as well)